tests/data/profiler_python_schema.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Simple json schema that regulates the output python dict format.

Reply:
not sure if the schema should be put in this path.

we should also mention this schema in the cpp implementation.